mindmeld/core.py

```diff
@@ -37,6 +37,8 @@
     "milliseconds": 0,
 }
 
+WHITESPACE_CHAR = ' '
```

Comment:
Perhaps we should be robust other types of whitespace (tabs, line breaks)

Reply:
To make this more robust we can use `string.whitespace` from the `string` library.

`"In Python3, string.whitespace is a pre-initialized string used as string constant. In Python, string.whitespace will give the characters space, tab, linefeed, return, formfeed, and vertical tab."`

[Source](https://www.geeksforgeeks.org/string-whitespace-in-python/#:~:text=whitespace%20is%20a%20pre%2Dinitialized,%2C%20formfeed%2C%20and%20vertical%20tab.&text=Parameters%20%3A%20Doesn't%20take%20any,since%20it's%20not%20a%20function.)

Reply:
Great points. @KKsharma99 @serapio  Do we have to worry about what we define as a "token" here when indexing these tokens? For example, for non-whitespace token delims in east asian languages, this method of finding "empty space" chars wouldnt work.

Reply:
I implemented a way of accounting for token spans based on the normalized tokens, that should be resilient to non-space tokenizers. Thoughts?

Reply:
That's a good point. Using the normalized tokens is definitely more robust here as compared to a whitespace delimiter. This should work with the upcoming changes of `character tokenization` and `spacy tokenization` as each individual token can be found in `query._normalized_tokens` as you have in your PR.